src/framework/ui/view/uitheme.cpp

```diff
@@ -752,59 +766,69 @@ int UiTheme::styleHint(QStyle::StyleHint hint, const QStyleOption* option, const
         } else {
-            drawRoundedRect(painter, rect, 3, backgroundColor, NO_BORDER);
+            drawRoundedRect(painter, rect, DEFAULT_RADIUS, backgroundColor, NO_BORDER);
         }
     } else {
-        drawRoundedRect(painter, rect, 3, backgroundColor, NO_BORDER);
+        drawRoundedRect(painter, rect, DEFAULT_RADIUS, backgroundColor, NO_BORDER);
+    }
+
+    if (styleState.focused) {
+        drawRoundedRect(painter, rect, DEFAULT_RADIUS, backgroundColor, QPen(fontPrimaryColor(), navCtrlBorderWidth()));
     }
 }
 
-void UiTheme::drawCheckboxIndicator(QPainter* painter, const QRect& rect, bool enabled, bool hovered, bool pressed, bool checked,
-                                    bool indeterminate, bool inMenu) const
+void UiTheme::drawCheckboxIndicator(QPainter* painter, const QRect& rect, const StyleState& styleState, bool checked, bool indeterminate,
+                                    bool inMenu) const
 {
+    QColor backgroundColor = buttonColor();
+    const qreal borderRadius = 2;
+
     if (!inMenu) {
-        QColor backgroundColor = buttonColor();
-        backgroundColor.setAlphaF(!enabled ? buttonOpacityNormal() * itemOpacityDisabled()
-                                  : pressed ? buttonOpacityHit()
-                                  : hovered ? buttonOpacityHover()
+        backgroundColor.setAlphaF(!styleState.enabled ? buttonOpacityNormal() * itemOpacityDisabled()
+                                  : styleState.pressed ? buttonOpacityHit()
+                                  : styleState.hovered ? buttonOpacityHover()
                                   : buttonOpacityNormal());
 
         QColor penBorderColor(Qt::transparent);
         int penBorderWidth = 0;
         if (configuration()->isHighContrast()) {
             penBorderColor = strokeColor();
-            penBorderWidth = enabled ? borderWidth() : 0;
+            penBorderWidth = styleState.enabled ? borderWidth() : 0;
         } else {
-            penBorderWidth = enabled && (hovered || pressed) ? borderWidth() : 0;
+            penBorderWidth = styleState.enabled && (styleState.hovered || styleState.pressed) ? borderWidth() : 0;
         }
 
-        penBorderColor.setAlphaF(pressed ? buttonOpacityHit() : hovered ? buttonOpacityHover() : buttonOpacityNormal());
-        drawRoundedRect(painter, rect, 2, backgroundColor, QPen(penBorderColor, penBorderWidth));
+        penBorderColor.setAlphaF(styleState.pressed ? buttonOpacityHit() : styleState.hovered ? buttonOpacityHover() : buttonOpacityNormal());
+        drawRoundedRect(painter, rect, borderRadius, backgroundColor, QPen(penBorderColor, penBorderWidth));
+    }
+
+    if (focused) {
+        drawRoundedRect(painter, rect, borderRadius, backgroundColor, QPen(fontPrimaryColor(), navCtrlBorderWidth()));
     }
 
     if (checked || indeterminate) {
         QColor tickColor = fontPrimaryColor();
-        if (!enabled) {
+        if (!styleState) {
```

Comment:
Needs to be `if (!styleState.enabled) {`

Reply:
Oh, thanks!
fixed